src/TLuaInterpreter.cpp

```diff
@@ -3579,6 +3579,44 @@ int TLuaInterpreter::setTextFormat( lua_State * L )
     return 1;
 }
 
+int TLuaInterpreter::raiseWindow( lua_State *L )
+{
+    string luaSendText="";
```

Comment:
This is a horrible case of repeated copy-paste-modify (probably originally from the `Send` command?) - why not call it `windowName` and make it a `QString` so that lower down we use `windowName = QString::fromUtf8( lua_tostring( L, 1 ) );` to get the value and there is no need for another intermediate `text`. You can see this being done in the `setTextFormat` function.